src/schema/v2/user.ts

```diff
@@ -43,6 +43,23 @@ export const UserAdminNoteType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLString),
     },
     createdAt: date(({ created_at }) => created_at),
+    creator: {
+      description: "The user who created the note",
+      type: UserType,
+      resolve: ({ created_by }, {}, { userByIDLoader }) => {
```

Comment:
Would also like to clarify if there is a difference between passing `{}` vs `_args` vs `_` when no args are getting passed into the `resolve`. I see all three forms used across the codebase. Do we prefer one over the other(s)?

Reply:
I would probably use `_args`, but I don't think there's a difference, they're all unused after-all.

Reply:
Thanks changed to your prefered one: https://github.com/artsy/metaphysics/pull/4515/commits/cf8ee33454914507cbc7d0da96eb9386f592d8d2